src/main/java/it/trade/api/TradeItApiClient.java

```diff
@@ -28,30 +29,45 @@
     protected String sessionToken;
     protected TradeItEnvironment environment;
     protected String apiKey;
+    protected boolean forceTLS12;
 
     public TradeItApiClient(String apiKey, TradeItEnvironment environment) {
-        this(apiKey, environment, null);
+        this(apiKey, environment, null, false);
+    }
+
+    public TradeItApiClient(String apiKey, TradeItEnvironment environment, boolean forceTLS12) {
+        this(apiKey, environment, null, forceTLS12);
     }
 
     /**
      * Use this constructor if you want to add your custom headers to each request
      */
     public TradeItApiClient(String apiKey, TradeItEnvironment environment, Interceptor requestInterceptor) {
+        this(apiKey, environment, requestInterceptor, false);
+    }
+
+    public TradeItApiClient(String apiKey, TradeItEnvironment environment, Interceptor requestInterceptor, boolean forceTLS12) {
         this.environment = environment;
         this.apiKey = apiKey;
         TradeItRequestWithKey.API_KEY = apiKey;
-        this.tradeItApi = createTradeItApi(environment, requestInterceptor);
+        this.tradeItApi = createTradeItApi(environment, requestInterceptor, forceTLS12);
     }
 
-    protected TradeItApi createTradeItApi(TradeItEnvironment environment, final Interceptor requestInterceptor) {
+    protected TradeItApi createTradeItApi(TradeItEnvironment environment, final Interceptor requestInterceptor, boolean forceTLS12) {
         OkHttpClient.Builder httpClientBuilder = new OkHttpClient().newBuilder();
+
         httpClientBuilder.connectTimeout(30, TimeUnit.SECONDS);
         httpClientBuilder.readTimeout(30, TimeUnit.SECONDS);
 
         if (requestInterceptor != null) {
             this.requestInterceptor = requestInterceptor;
             httpClientBuilder.networkInterceptors().add(requestInterceptor);
         }
+
+        if (forceTLS12) {
+            httpClientBuilder = Tls12SocketFactory.enableTls12(httpClientBuilder);
```

Comment:
No, it just sets some additional configuration on the same `httpClientBuilder`.
I guess we could just return void and just call:
`Tls12SocketFactory.enableTls12(httpClientBuilder);`